.github/workflows/build-macos.yml

```
@@ -14,9 +14,9 @@ jobs:
     runs-on: macos-latest
     steps:
     - name: Checkout code
-      uses: actions/checkout@v6
+      uses: actions/checkout@d23441a48e516b6c34aea4fa41551a30e30af803 # v6
     - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.12)

[error] property "node-version" is not defined in object type {}

(expression)